tests/base/test_regularization.py

```diff
@@ -626,7 +626,7 @@ def test_cross_ref_reg(dim):
     # give it some cell weights, and some cell vector weights to do something with
     cell_weights = np.random.rand(n_active)
     cell_vec_weights = np.random.rand(n_active, dim)
-    cross_reg.set_weights(cell_weights=cell_weights)
+    cross_reg.set_weights(weights=cell_weights)
```

Comment:
Just a test, but this was fine as is, as it made it clear they were cell weights in the dictionary.

Reply:
Totally agree. I shouldn't have changed that one.